tinode-db/data.json

```diff
@@ -244,6 +245,14 @@
    "private": {"comment": "I'm not the owner, Frank is"},
    "topic": "*BF",
    "user": "bob"
+  },
+  {
+   "createdAt": "-126h",
+   "name": "support",
+   "owner": "frank",
+   "tags": ["support","public"],
+   "public": {"fn": "Support", "photo": "support-64.jpg", "type": "jpg"},
+   "access": {"anon": 64, "auth": 64}
```

Comment:
This is a good idea but tinode-db/makedb.go must be updated too. Otherwise it's not going to be used.

Reply:
I commited changes to go file... but I don't know how to easily run it to test... what cmd line do you use to build and test makedb?